src/garage/tf/policies/gaussian_mlp_policy.py

```diff
@@ -142,21 +142,39 @@ def __init__(self,
             layer_normalization=layer_normalization,
             name='GaussianMLPModel')
 
+        self._initialize()
+
+    def _initialize(self):
+        """Initialize policy."""
+        with tf.compat.v1.variable_scope(self.name) as vs:
+            self._variable_scope = vs
+            state_input = tf.compat.v1.placeholder(tf.float32,
+                                                   shape=(None, None,
+                                                          self._obs_dim))
+            self._dist, mean, log_std = self.model.build(state_input).outputs
+            self._f_dist = tf.compat.v1.get_default_session().make_callable(
+                [self._dist.sample(), mean, log_std], feed_list=[state_input])
+
+    @property
+    def input_dim(self):
+        """int: Dimension of the policy input."""
+        return self._obs_dim
+
     def build(self, state_input, name=None):
-        """Build model.
+        """Build policy.
 
         Args:
-          state_input (tf.Tensor): State input.
-          name (str): Name of the model, which is also the name scope.
+            state_input (tf.Tensor) : State input.
+            name (str): Name of the policy, which is also the name scope.
+
+        Returns:
```

Comment:
The docstring of returns is wrong - `build()` returns a namedtuple, not a tuple.

Reply:
ah, probably same for all the policies since they all now returned a `network` namedtuple. I'll fix them all together in another PR.